docs/proposals/scheduling/20220510-load-aware-scheduling.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Load Aware Scheduling
+authors:
+  - "@eahydra"
+reviewers:
+  - "@hormes"
+creation-date: 2022-05-10
+last-updated: 2022-05-13
+status: provisional
+---
+
+# Load Aware Scheduling
+
+## Summary
+
+Although Koordinator provides the co-location mechanism to improve the resource utilization of the cluster and reduce costs, it does not yet have the ability to control the utilization level of the cluster dimension. This proposal defines a scheduling plugin to help Koordinator achieve this capability.
+
+## Motivation
+
+Koordinator oversells some resources through the co-location mechanism. Although it can improve the utilization of nodes, Best Effort workloads may also interfere with latency-sensitive applications. 
+
+### Goals
+
+1. Provides a configurable scheduling plugin to help control cluster resource utilization.
+2. Utilization control mechanism that supports multiple resources.
+3. Control resource utilization at a safe threshold.
+
+### Non-Goals/Future Work
+1. Guide scheduling through application profiles. This is left as a follow up work that will be done under a different proposal.
+
+## Proposal
+
+### User stories
+
+#### Story 1
+
+When the resource utilization of the node has reached a high threshold, serious resource contention will occur between the running workloads on the node. For example, best effort workloads are frequently suppressed due to higher-priority applications requiring resources.As a result, best effort workloads are timed out or even forced to end; or a latency-sensitive application will suffer severe performance degradation under high utilization, failing to meet external SLAs. This should be avoided.
+
+#### Story 2
+
+Workloads in a co-located cluster have different resource requirements. Typical CPU-bound workloads expect to use more CPU, while other types of workloads may use more memory. If only pay attention to the utilization threshold of a certain resource, it may cause the utilization of another resource to be low or high, which will lead to the insufficient use of resources in the global dimension.
```

Comment:
This story confuses users and can be focused on Prod Pods should focus on non-BE utilization.